config/server.lua

```diff
@@ -1,23 +1,23 @@
 return {
     commandPerms = {
-        useMenu = 'group.mod',
-        noclip = 'group.mod',
-        names = 'group.mod',
-        blips = 'group.mod',
-        setModel = 'group.admin',
-        dev = 'group.admin',
+        useMenu = 'mod',
+        noclip = 'mod',
+        names = 'mod',
+        blips = 'mod',
+        setModel = 'admin',
+        dev = 'admin',
     },
 
     eventPerms = {
-        playerOptionsGeneral = 'group.mod',
-        kick = 'group.mod',
-        ban = 'group.admin',
-        changePerms = 'group.admin',
-        changePlayerData = 'group.admin',
-        giveAllWeapons = 'group.admin',
-        getRadioList = 'group.mod',
-        useMenu = 'group.mod',
-        clothingMenu = 'group.admin',
+        playerOptionsGeneral = 'mod',
+        kick = 'mod',
+        ban = 'admin',
+        changePerms = 'admin',
+        changePlayerData = 'admin',
+        giveAllWeapons = 'admin',
+        getRadioList = 'mod',
+        useMenu = 'mod',
+        clothingMenu = 'admin',
```

Comment:
These should be changed back to group.* since we're now using the native.

Reply:
Looking at https://github.com/Qbox-project/qbx_core/blob/main/server/functions.lua#L273 I don't see any code that would cutoff the prefix. It looks like just pass through to the native. Where are you getting that 'group.' is being stripped?

Reply:
> Looking at https://github.com/Qbox-project/qbx_core/blob/main/server/functions.lua#L273 I don't see any code that would cutoff the prefix. It looks like just pass through to the native. Where are you getting that 'group.' is being stripped?

I did a lib.print of the output from `exports.qbx_core:HasPermission` and it had a table of keys that just showed mod or admin with no group. in front

Reply:
the output is a boolean though? What did you print?

Reply:
Oh yeah, GetPermission cuts it off, but HasPermission does not. We should probably deprecate those two exports.